Add ReadyToRun environment variable run-time option (#16834)
dotnet/docs: docs/core/run-time-config/compilation.md

</Project>
```

## ReadyToRun

> - Configures whether the .NET Core runtime uses pre-compiled images for framework code (and user assemblies, if pre-compiled images are available). Disabling this option forces the runtime to JIT-compile framework code.

Author: Is the part about user assemblies true, or are the R2R images always used if they're available?

Reviewer: There is no distinction between user/framework assemblies for this variable

> - Configures whether the .NET Core runtime uses pre-compiled images for framework code (and user assemblies, if pre-compiled images are available). Disabling this option forces the runtime to JIT-compile framework code.

Reviewer: ```suggestion
- Configures whether the .NET Core runtime uses pre-compiled code for images with available ReadyToRun data. Disabling this option forces the runtime to JIT-compile framework code.
```

> - For more information see [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images).

Author: ```suggestion
- For more information, see [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images).
```

- Default: Enabled (`1`).

| | Setting name | Values |
| - | - | - |

> | **Environment variable** | `COMPlus_ReadyToRun` | `1` - enabled<br/>`0` - disabled |

Reviewer: Can we clarify that `1` is the default?

Author: It is clarified in the bulleted list. Is that okay? That's how we've done it for the other options too.

Reviewer: Ah! didn't notice :). Thanks
